app/mailers/user_update_mailer.rb

```diff
@@ -8,6 +8,7 @@ def new_profile_email(person, by_email = nil)
     @person = person
     @by_email = by_email
     @profile_url = profile_url(person)
+    @firefox_browser_warning = t('.firefox_message', default: '')
```

Comment:
but test this works using mailer previews first as i have not checked this works